### ISSUE TITLE
Updates needed for co-maintainer status

### ISSUE THREAD

**Issue by md5** [appropriate/docker-jetty#19]
Now that @gregw is helping maintain this image, we should update the `LICENSE` to correctly reflect the copyright situation (I'm sure that would make some lawyers at the Eclipse Project happy). I also noticed we don't have `MAINTAINER` in the `Dockerfile`, so that should be added as well.

**Comment by gregw** [comment]
From the Jetty NOTICE.TXT file:

```
The Jetty Web Container is Copyright Mort Bay Consulting Pty Ltd
unless otherwise noted.

Jetty is dual licensed under both

  * The Apache 2.0 License
    http://www.apache.org/licenses/LICENSE-2.0.html

      and

  * The Eclipse Public 1.0 License
    http://www.eclipse.org/legal/epl-v10.html

Jetty may be distributed under either license.
```

So distributing under apache 2.0 is fine.

In terms of copyright, I'm happy to assign copyright of any contributions to whomever/whatever holds the current copyright if the project.

**Comment by md5** [comment]
The copyright is currently assigned to me, since I've done all the work until recently. My only reasons to think that the copyright might need updating is that there's no formal agreement to assign copyright to me and because this image will eventually be maintained by the Jetty Project.

Perhaps we can just cross that bridge when we come to it.

Thanks for confirming that Apache 2.0 is fine. I checked Jetty's license when I started the image and saw the dual licensing, but I though a plain Apache 2.0 license was easier. :+1:
